sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -66,8 +72,11 @@ public TimeSpan MessageWaitTimeout
             }
         }
 
-        /// <summary>Gets or sets the maximum number of existing sessions that the User wants to handle concurrently.</summary>
-        /// <value>The maximum number of sessions that the User wants to handle concurrently.</value>
+        /// <summary>Gets or sets the maximum number of existing sessions that the User wants to handle concurrently. Setting this value to be 
+        /// greater than the max number of active sessions in the service will not increase message throughput. The backend implementation is once the 
```

Comment:
This does convey the idea. Wonder if it could be reworded slightly better.
"backend" would be misleading, and could mean the service. "Background" might be a better word. 
In the remarks, we could probably write something along the lines of:
"The session-pump (SDK) will accept `MaxConcurrentSessions` number of sessions in parallel and dispatch the messages. The messages within a session are delivered sequentially. If more than <cref="MaxConcurrentSessions"/> number of sessions are present in the entity, they will be accepted one-by-one after closing the existing sessions".